modules/openapi-generator/src/main/resources/cpp-qt5-client/model-body.mustache

```diff
@@ -22,24 +21,19 @@ namespace {{this}} {
     this->initializeModel();
 }
 
-{{classname}}::~{{classname}}() {
+{{classname}}::~{{classname}}() {}
 
-}
-
-void
-{{classname}}::initializeModel() {
-    {{^isEnum}}{{#vars}}
+void {{classname}}::initializeModel() {{=<% %>=}}{<%={{ }}=%>{{^isEnum}}{{#vars}}
```

Comment:
what is this change for?

Reply:
This (kind of dirty) trick allowed me to replace temporaly the template delimiter from `{{` / `}}` to `<%` / `%>`.

My first iteration was:

```
void {{classname}}::initializeModel() {{{^isEnum}}{{#vars}}
    m_{{name}}_isSet = false;
    ...
```

But if fails since mustache didn't like the `{{{`.

Reply:
you can give a space.

Reply:
I will rather give a carriage return to avoid extra space at the end of the line: a84a0038c6